broker-client/src/main/java/org/aktin/broker/client/BrokerAdmin.java

```diff
@@ -103,10 +153,30 @@ public interface BrokerAdmin {
 
 	String getResultString(int requestId, int nodeId) throws IOException;
 	ResponseWithMetadata getResult(int requestId, int nodeId) throws IOException;
-
+	
+	/**
+	 * Make sure, the Request is actually targeted (see {@link RequestInfo}), as otherwise an exception is thrown.
```

Comment:
Again, documentation missing. Comments only.
Exception for un-targeted requests sounds like a bug -> create issue

Reply:
This conflicts with the test for untargetedRequest.. see review comment there